### PR TITLE
pixi 0.16.0

### PR DESCRIPTION
**Pull request by pavelzw**
Created with `brew bump-formula-pr`.<details>
  <summary>release notes</summary>
  <pre>### ✨ Highlights
- This release removes [`rip`]() and add [`uv`]() as the PyPI resolver and installer.

#### Added
- Add tcsh install support by @obust in [#898](https://github.com/prefix-dev/pixi/pull/898)
- Add user agent to pixi http client by @baszalmstra in [#892](https://github.com/prefix-dev/pixi/pull/892)
- Add a schema for the pixi.toml by @ruben-arts in [#936](https://github.com/prefix-dev/pixi/pull/936)

#### Changed
- Switch from rip to uv by @tdejager in [#863](https://github.com/prefix-dev/pixi/pull/863)
- Move uv options into context by @tdejager in [#911](https://github.com/prefix-dev/pixi/pull/911)
- Add Deltares projects to Community.md by @Hofer-Julian in [#920](https://github.com/prefix-dev/pixi/pull/920)
- Upgrade to uv 0.1.16, updated for changes in the API by @tdejager in [#935](https://github.com/prefix-dev/pixi/pull/935)

#### Fixed
- Made the uv re-install logic a bit more clear by @tdejager in [#894](https://github.com/prefix-dev/pixi/pull/894)
- Avoid duplicate pip dependency while importing environment.yaml by @sumanth-manchala in [#890](https://github.com/prefix-dev/pixi/pull/890)
- Handle custom channels when importing from env yaml by @sumanth-manchala in [#901](https://github.com/prefix-dev/pixi/pull/901)
- Pip editable installs getting uninstalled by @renan-r-santos in [#902](https://github.com/prefix-dev/pixi/pull/902)
- Highlight pypi deps in pixi list by @sumanth-manchala in [#907](https://github.com/prefix-dev/pixi/pull/907)
- Default to the default environment if possible by @ruben-arts in [#921](https://github.com/prefix-dev/pixi/pull/921)
- Switching channels by @baszalmstra in [#923](https://github.com/prefix-dev/pixi/pull/923)
- Use correct name of the channel on adding by @ruben-arts in [#928](https://github.com/prefix-dev/pixi/pull/928)
- Turn back on jlap for faster repodata fetching by @ruben-arts in [#937](https://github.com/prefix-dev/pixi/pull/937)
- Remove dists site-packages's when python interpreter changes by @tdejager in [#896](https://github.com/prefix-dev/pixi/pull/896)

## New Contributors
* @obust made their first contribution in [#898](https://github.com/prefix-dev/pixi/pull/898)
* @renan-r-santos made their first contribution in [#902](https://github.com/prefix-dev/pixi/pull/902)

[Full Commit history](https://github.com/prefix-dev/pixi/compare/v0.15.2..v0.16.0)</pre>
</details>

### CODE REVIEW

**Comment by pavelzw** [comment]
```
==> brew linkage --test pixi
==> FAILED
Full linkage --test pixi output
  Broken dependencies:
    /opt/homebrew/opt/openssl@3/lib/libcrypto.3.dylib (openssl@3)
    /opt/homebrew/opt/openssl@3/lib/libssl.3.dylib (openssl@3)
Truncated linkage --test pixi output
  Error: Broken dependencies:
    /opt/homebrew/opt/openssl@3/lib/libcrypto.3.dylib (openssl@3)
    /opt/homebrew/opt/openssl@3/lib/libssl.3.dylib (openssl@3)
  
==> brew linkage --cached pixi
==> brew install --only-dependencies --include-test pixi
==> brew test --verbose pixi
==> FAILED
Full test pixi output
  /opt/homebrew/Library/Homebrew/vendor/bundle/ruby/3.1.0/bin/bundle clean
  ==> Testing pixi
  ==> /opt/homebrew/Cellar/pixi/0.16.0/bin/pixi --version
  dyld[88[59](https://github.com/Homebrew/homebrew-core/actions/runs/8215631869/job/22469325333?pr=165621#step:3:62)]: Library not loaded: '/opt/homebrew/opt/openssl@3/lib/libssl.3.dylib'
    Referenced from: '/opt/homebrew/Cellar/pixi/0.16.0/bin/pixi'
    Reason: tried: '/opt/homebrew/opt/openssl@3/lib/libssl.3.dylib' (no such file), '/usr/local/lib/libssl.3.dylib' (no such file), '/usr/lib/libssl.3.dylib' (no such file)
```

**Comment by pavelzw** [comment]
```
==> brew linkage --test pixi
==> FAILED
Full linkage --test pixi output
  Missing libraries:
    libgit2.so.1.7
Truncated linkage --test pixi output
  Error: Missing libraries:
    libgit2.so.1.7
  
==> brew linkage --cached pixi
==> brew install --only-dependencies --include-test pixi
==> brew test --verbose pixi
==> FAILED
Full test pixi output
  /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.1.0/bin/bundle clean
  ==> Testing pixi
  ==> /home/linuxbrew/.linuxbrew/Cellar/pixi/0.16.0/bin/pixi --version
  /home/linuxbrew/.linuxbrew/Cellar/pixi/0.16.0/bin/pixi: error while loading shared libraries: libgit2.so.1.7: cannot open shared object file: No such file or directory
  
  Error: pixi: failed
```

File: Formula/p/pixi.rb [hunk 1]
```diff
@@ -25,24 +25,48 @@ class Pixi < Formula
...
+
+    system bin/"pixi", "init"
     assert_path_exists testpath/"pixi.toml"
+
+    linked_libraries = [
+      Formula["libgit2"].opt_lib/shared_library("libgit2"),
+      Formula["openssl@3"].opt_lib/shared_library("libssl"),
+      Formula["openssl@3"].opt_lib/shared_library("libcrypto"),
+    ]
+    linked_libraries << (Formula["openssl@3"].opt_lib/shared_library("libcrypto")) if OS.mac?
```
Comment (ZhongRuoyu) [inline Formula/p/pixi.rb:66]:
```suggestion
```

**Review by ZhongRuoyu — APPROVED** [review]

**Comment by ZhongRuoyu** [comment]
Thanks @pavelzw and @chenrui333!

**Pull request merged**